src/Amazon.AspNetCore.Identity.Cognito/CognitoUserStore/CognitoUserStore.cs

```diff
@@ -227,6 +228,31 @@ public virtual async Task<IdentityResult> ResetPasswordAsync(TUser user, Cancell
             }
         }
 
+        /// <summary>
+        /// Queries Cognito and returns all the users in the pool.
+        /// </summary>
+        /// <returns>
+        /// The <see cref="Task"/> that represents the asynchronous operation, containing a IEnumerable of CognitoUser.
+        /// </returns>
+        public virtual async Task<IEnumerable<CognitoUser>> GetUsersAsync(CancellationToken cancellationToken)
+        {
+            // This API is not paginated.
```

Comment:
This API is paginated, no?  https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_ListUsers.html